dashboard/app/models/script_level.rb

```diff
@@ -67,6 +67,16 @@ def end_of_stage?
     stage.script_levels.to_a.last == self
   end
 
+  def long_assessment?
+    if self[:assessment]
+      level_info = Level.find(self[:level_id])
```

Comment:
You can just use `level` here (internally does almost the same thing -- level is a method that calls Level.find(self[:level_id] and caches it in the instance --  one advantage of using the method is that you don't need to find it again if it has already been called, which is likely)